sklearn/feature_selection/tests/test_sequential.py

```diff
@@ -105,7 +198,9 @@ def test_sparse_support():
 
     X, y = make_regression(n_features=10)
     X = scipy.sparse.csr_matrix(X)
-    sfs = SequentialFeatureSelector(LinearRegression(), cv=2)
+    sfs = SequentialFeatureSelector(
+        LinearRegression(), n_features_to_select="auto", cv=2
+    )
```

Comment:
We still need to test `None` here. We can do this by parameterizing `test_sparse_support` to go through `None` and `'auto'`

```python
# TODO: Remove test for n_features_to_select=None in 1.2
@pytest.mark.filterwarnings("ignore:Leaving `n_features_to_select` to ")
@pytest.mark.parametrize("n_features_to_select", ["auto", None])
def test_sparse_support(n_features_to_select):
        ...
        sfs = SequentialFeatureSelector(
            LinearRegression(), n_features_to_select=n_features_to_select, cv=2
        )
```

Reply:
add `None` with https://github.com/scikit-learn/scikit-learn/pull/20145/commits/0dd1499a646bcfa47fadf2d3ca2db9e8329e54ee